cmd/pitr/collector/collector.go

```diff
@@ -411,6 +458,9 @@ func (c *Collector) CollectBinLogs(ctx context.Context) error {
 			return errors.Wrap(err, "update timeline file")
 		}
 	}
+
+	pxcBinlogCollectorLastUploadTime.SetToCurrentTime()
```

Comment:
I think we should move this either at the end of `c.manageBinlog` function or line 451